packages/react-native-app-auth/android/src/main/java/com/rnappauth/RNAppAuthModule.java

```diff
@@ -1088,6 +1088,10 @@ private BrowserMatcher getBrowserAllowList(ReadableArray androidAllowCustomBrows
                     browserMatchers.add(VersionedBrowserMatcher.SAMSUNG_CUSTOM_TAB);
                     break;
                 }
+                case "edge": {
+                    browserMatchers.add(VersionedBrowserMatcher.EDGE_CUSTOM_TAB);
```

Comment:
When I run `npx react-native run-android` I get the following:
```bash
error: cannot find symbol browserMatchers.add(VersionedBrowserMatcher.EDGE_CUSTOM_TAB);
```
As per these [docs](https://www.javadoc.io/doc/net.openid/appauth/latest/net/openid/appauth/browser/VersionedBrowserMatcher.html)

However, when I comment this out, and set my default browser to Edge in the Android Emulator, the Example app does successfully open the links in Edge.